### PR TITLE
updated composer

### LINKED ISSUE #34: cannot update to 3.4.0. on api.
composer require dotkernel/dot-errorhandler 3.4.0
./composer.json has been updated
Running composer update dotkernel/dot-errorhandler
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires dotkernel/dot-errorhandler 3.4.0 -> satisfiable by dotkernel/dot-errorhandler[3.4.0].
    - dotkernel/dot-errorhandler 3.4.0 requires psr/http-message ^2.0 -> found psr/http-message[2.0] but the package is fixed to 1.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

### PR DESCRIPTION
**Pull request by bidi47**
https://github.com/dotkernel/dot-errorhandler/issues/34

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "laminas/laminas-dependency-plugin": false
```
Comment (alexmerlin) [inline composer.json]:
```suggestion
```
Didn't we already remove this?

File: composer.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
```
Comment (alexmerlin) [inline composer.json]:
```suggestion
            "dealerdirect/phpcodesniffer-composer-installer": true
```

**Review by alexmerlin — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (bidi47) [reply composer.json]: a phpunit check failed related to this, so i had to update 'allow-plugins'
https://github.com/dotkernel/dot-errorhandler/actions/runs/9159744416/job/25180860644

Comment (bidi47) [reply composer.json]: removed in the latest commit
along with php 8.1

**Review by alexmerlin — APPROVED** [review]

**Pull request merged**